app/controllers/volunteers_controller.rb

```diff
@@ -0,0 +1,32 @@
+class VolunteersController < ApplicationController
+  before_action :set_volunteer, only: %i[destroy]
+  before_action :find_post, only: %i[create destroy]
+
+  def create
+    @volunteer = @post.volunteers.create(volunteer_params)
+    @volunteer.user = current_user
```

Comment:
merge this user param in volunteer_params, and create it.

Reply:
Don't understand. Could you please explain?